ui/app/src/Views/SilenceForm/Views.elm

```diff
@@ -44,9 +45,52 @@ view maybeId { matchers, comment } defaultCreator { form, silenceId, alerts, act
             [ informationBlock activeAlertId silenceId alerts
             , silenceActionButtons maybeId form resetClick
             ]
+        , dateTimePickerDialog form
         ]
 
 
+dateTimePickerDialog : SilenceForm -> Html SilenceFormMsg
+dateTimePickerDialog form =
+    case form.viewDateTimePicker of
+        True ->
+            div []
+                [ div [ class "modal fade show", style "display" "block" ]
+                    [ div [ class "modal-dialog modal-dialog-centered" ]
+                        [ div [ class "modal-content" ]
+                            [ div [ class "modal-header" ]
+                                [ button
+                                    [ class "close ml-auto"
+                                    , onClick (CloseDateTimePicker |> UpdateField)
+                                    ]
+                                    [ text "x" ]
+                                ]
+                            , div [ class "modal-body" ]
+                                [ viewDateTimePicker dateTimePickerConfig form.dateTimePicker ]
```

Comment:
If we use `utc`, then `viewDateTimePicker` could take `UpdateDateTimePicker` and the `dateTimePickerConfig` would not be needed.

Reply:
Thank's. I removed ...Config.